olm-catalog/serverless-operator/manifests/operator_v1alpha1_knativekafka_crd.yaml

```diff
@@ -45,6 +45,14 @@ spec:
                     description: Enabled defines if the KafkaChannel installation
                       is enabled
                     type: boolean
+                  authSecretNamespace:
+                    description: AuthSecretNamespace is the namespace of the secret that contains Kafka
+                      auth configuration.
+                    type: string
+                  authSecretName:
+                    description: AuthSecretName is the name of the secret that contains Kafka
+                      auth configuration.
+                    type: string
```

Comment:
It seems like a common idiom for kafka auth is having the tls auth in one secret, and the sasl auth in another.  How would this handle that? If they're both used for the same kafka producer/consumer config, the tls is used for encryption but sasl for auth.

Reply:
@lberk here is a reference to the script https://github.com/knative-sandbox/eventing-kafka/blob/master/test/e2e-tests.sh#L363-L373

(based on my e-mail reply before)